### PR TITLE
docs - events and timelines

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Documents events and timelines.

### CODE REVIEW

File: docs/users-guide/events-and-timelines.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+You can group events into timelines. For example, you may want to have an email campaign timeline that tracks, or an outages timeline that tracks downtime.
+
+## How collections, timelines, and events fit together
+
+Collections have timelines, and timelines contain events. 
+
+### Moving events to other timelines 
+
+TOOD: Currently, you can't "cross timelines". We're working out the physics behind timeline portals, but in the meantime, if you want to move an event from one timeline to another, you'll need to archive the original event, and recreate that event in the new/other timeline.
```
Comment (mazameli) [inline docs/users-guide/events-and-timelines.md:36]: We may have finished implementing moving events to other timelines/collections by Moday.

**Review by mazameli — COMMENTED** [review]
Are you going to be calling out somewhere else that you can add/edit events from Simple Mode, and that you can display them on unsaved/adhoc charts too?

File: docs/users-guide/events-and-timelines.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Events and timelines
+
+A lot of discussions around data have a moment when someone asks a question related to a specific point in time: "Wait, what's the spike in March again?", or "When did the new widget launch?"
+
+Events and timelines are a way to capture that chronological knowledge and make it available when you need it, in context (that is, when you're viewing a chart). Events are a great way to store institutional knowledge about what happened and when, so people (including yourself three months from now) won't have to figure out (again) why the line chart spiked back in March.
+
+## Events
+
+Events are basically dates + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
```
Comment (frannietrempe) [inline docs/users-guide/events-and-timelines.md:9]: Can events span a date range? "Dates" is plural here but title, description, and icon are singular, so it's a little unclear from reading

File: docs/users-guide/events-and-timelines.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Timelines
+
+You can group events into timelines, and associate those timelines with [collections](collections.md). For example, you may want to have a timeline that contains important email or sales dates, or an outages timeline that tracks downtime. You can move events between timelines, and move timelines from collection to collection.
+
+Collections can have timelines, and timelines can contain events. When we say, "collections have timelines", in practice what this means is that when you view a time series question in that collection, you'll see events from all the collection's timelines plotted on its chart. Events are invisible on dashboards.
```
Comment (frannietrempe) [inline docs/users-guide/events-and-timelines.md:15]: Something about the structure of this paragraph is confusing to me - maybe the "time series question in _that_ collection" <-- what is "that" referring to? Tried a couple different ways to rewrite it but nothing was sticking. Can it be rephrased or switched around somehow? Is this close to it?:

"Collections can have timelines, and timelines can contain events. In practice what this means is that events you've added to a timeline will show up on time series questions stored in the same collection as that timeline. Events are invisible on dashboards."

File: docs/users-guide/events-and-timelines.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+### Adding events when viewing a collection
+
+When viewing a [collection](collections.md), you can view, add, or edit events by clicking on the calendar icon in the upper right.
```
Comment (frannietrempe) [inline docs/users-guide/events-and-timelines.md:19]:
```suggestion
When viewing a [collection](collections.md), you can view, add, or edit events by clicking on the **calendar** icon in the upper right.
```

**Review by frannietrempe — APPROVED** [review]
Looks good. The only bit that seems missing here is something about who can create and edit events, maybe how events/timelines intersect with permissions? Is event/timeline creation linked to the permissions of the collection they're in?

**Pull request merged**